plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergParquetSystemTables.java

```diff
@@ -22,4 +29,31 @@ public TestIcebergParquetSystemTables()
     {
         super(PARQUET);
     }
+
+    @Test
+    public void testPropertiesTable()
+    {
+        try (TestTable table = newTrinoTable("test_properties_table", "AS SELECT 1 x")) {
+            assertThat(query("SELECT * FROM \"%s$properties\"".formatted(table.getName())))
+                    .matches("""
+                            VALUES (VARCHAR 'write.format.default', VARCHAR 'PARQUET'),
+                            (VARCHAR 'commit.retry.num-retries', VARCHAR '4'),
+                            (VARCHAR 'write.parquet.compression-codec', VARCHAR 'zstd')""");
+        }
+        String tableName = "test_table_codec" + randomNameSuffix();
+        Session snappySession = Session.builder(getSession())
```

Comment:
you can do `ALTER TABLE .... SET PROPERTIES...` - maybe it is relevant for your test

Reply:
Thank you - adding a test case